scene/3d/mesh_instance_3d.h

```diff
@@ -97,6 +99,9 @@ class MeshInstance3D : public GeometryInstance3D {
 
 	MeshInstance3D();
 	~MeshInstance3D();
+
+private:
+    String default_collider_suffix = "_staticbody3D";
```

Comment:
Not sure, but I can imagine the old `_col` suffix was chosen intentionally to [fit the naming convention](https://docs.godotengine.org/en/latest/tutorials/assets_pipeline/importing_scenes.html#create-collisions-col-convcol-colonly-convcolonly) of importing collision meshes?